components/molecule/notification/src/index.js

```diff
@@ -38,6 +38,10 @@ class MoleculeNotification extends Component {
     this.state.show !== nextProps.show && this.toggleShow()
   }
 
+  componentDidMount() {
+    this.autoCloseIfRequired()
```

Comment:
What if show is `false` here. You're setting an unneeded timeout, right ?

Reply:
True. The fact that has a different behaviour when `show=true` or `show=false` has confused me again 🤣